### PR TITLE
Remove references to tensorflow in original forked code.

### PR DESCRIPTION
**Pull request by bparr**
Removes a bunch of code we do not plan on using. It will still be in version control if we change our mind.

Part of https://github.com/bparr/melee-ai/issues/10

### CODE REVIEW

File: ctype_util.py [hunk 1]
```diff
@@ -91,68 +89,3 @@ def randomValue(ctype):
   # TODO: handle arrays
   raise TypeError("Unsupported type %s" % ctype)
 
-# TODO: fill out the rest of this table
```
Comment (deepakdilipkumar) [inline ctype_util.py]: Any idea what the rest of this is for?

File: agent.py [hunk 2]
```diff
@@ -18,147 +15,43 @@ class Agent(Default):
...
-    
-    self.state = state
-    
+
     current = self.memory.peek()
     current.state = state
-    
-    self.prev_action = self.action
     current.prev_action = self.prev_action
+    current.action = action
```
Comment (deepakdilipkumar) [inline agent.py]: I don't get why current is needed at all. I tried removing all the lines involving current (action, prev_action, state) and it seems to work fine.

File: ctype_util.py (hunk 1 above)
Comment (deepakdilipkumar) [inline ctype_util.py]: Rest meaning the rest of the file which is now gone.

File: agent.py (hunk 2 above)
Comment (bparr) [inline agent.py]: It's a bit convoluted, but it is changing the return value of agent.act(), which gets returned in make_action() and by advance_frame(), and by env.step(). That's why I thought we needed it. Am I reading this wrong?

File: ctype_util.py (hunk 1 above)
Comment (bparr) [inline ctype_util.py]: Yes.

**Pull request merged**